server/src-rsr/initialise.sql

```diff
@@ -288,7 +288,8 @@ CREATE TABLE hdb_catalog.event_triggers
   configuration JSON,
   comment TEXT,
   FOREIGN KEY (schema_name, table_name)
-  REFERENCES hdb_catalog.hdb_table(table_schema, table_name) ON UPDATE CASCADE
+    REFERENCES hdb_catalog.hdb_table(table_schema, table_name)
+    ON UPDATE CASCADE DEFERRABLE INITIALLY DEFERRED
```

Comment:
shouldn't it be `INITIALLY IMMEDIATE` as per `server/src-rsr/migrations/40_to_41.sql`

Reply:
Would it be less noisy to just say `DEFERRABLE` ? After all, we are adding this migration just so that the `SET CONSTRAINTS` can only work for `DEFERRABLE` constraints. The default behaviour for DEFERRABLE feels sufficiently obvious.